src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/matchxpath/Example1.java

```diff
@@ -2,22 +2,24 @@
 <module name="Checker">
   <module name="TreeWalker">
     <module name="MatchXpath">
-      <property name="query" value="//METHOD_DEF[.//LITERAL_PRIVATE and
-              following-sibling::METHOD_DEF[.//LITERAL_PUBLIC]]"/>
+      <property name="query"
+       value="//METHOD_DEF[.//LITERAL_PRIVATE and
+       following-sibling::METHOD_DEF[.//LITERAL_PUBLIC]]"/>
```

Comment:
Please put 4 spaces on the left of `value=` to indent it more.
Move `and` next line .

Reply:
Done.